include/libtorrent/aux_/platform_util.hpp

```diff
@@ -1,11 +1,13 @@
 #ifndef TORRENT_PLATFORM_UTIL_HPP
 #define TORRENT_PLATFORM_UTIL_HPP
 
+#include "libtorrent/aux_/export.hpp"
+
 #include <cstdint>
 
-namespace libtorrent {
+namespace libtorrent::aux {
 
-	int max_open_files();
+	TORRENT_EXTRA_EXPORT int max_open_files();
```

Comment:
what prompted this? technically it's only necessary once there's a test that needs access to the function.
I think it's a good change, I'm just curious if you experienced an error without it.

Reply:
no error, was just mechanical thinking on my part